static/faq.html

```diff
@@ -47,7 +47,7 @@
 Filters file names according to their extension.<br>
 To find source code dealing with XMPP written in Perl, you could search for "<tt>XMPP
 filetype:perl</tt>".<br>
-The currently supported file types are c, c++, perl, python, go, java, ruby, shell, vala, javascript, json.
+The currently supported file types are c, objective-c, objective-c++, c++, perl, python, go, java, ruby, shell, vala, javascript, json.
```

Comment:
Here you list “objective-c” and “objective-c++”, but in the other file you introduce them as “objc” and “objc++”. I’d prefer the shorter names, so let’s change it to “objc” and “objc++” here.